src/pages/HomePage/pages/LifeCenter/components/LifeCenterCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
```

Comment:
to enhance tree shaking try and import only what you need 
{ReactNode} instead of React